### DISCUSSION TITLE
Looking for an alternative to complete

### DISCUSSION THREAD

**Discussion by wiiznokes** [nushell/nushell#14292] [Q&A]
Hi, i'm new to nushell. I'm doing a quick script to run some command. I would like a function similar to complete but
- it should redirect stdout & stderr directly to the terminal (in a stream fashion)
- return an int for exit_code

Does this function exist/feasible ?

**Comment by ghost** [comment]
im not entirely sure what exactly your goal here is, but if you just want the exitcode from a command you can use the `$env,LAST_EXIT_CODE` variable after the command finishes executing (which is a int).  

example:

```nu
git commit -m foo
if $env.LAST_EXIT_CODE != 0 { print "ERROR" }
```
